apps/src/p5lab/gamelab/GameLab.js

```diff
@@ -21,6 +21,11 @@ GameLab.prototype.init = function(config) {
     }));
   }
 
+  // Push initial level properties into the Redux store
+  this.studioApp_.setPageConstants(config, {
+    validationEnabled: !!config.level.validationEnabled
```

Comment:
Not sure where we landed on including this in sprite/dance lab, but if that's something we'd still like to do, this should go in P5Lab.js instead. This file is specific to GameLab, rather than the gamelab family of labs.

Reply:
ah, thanks! apparently the same inheritance from the ruby models doesn't apply!